.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -50,5 +62,9 @@ Steps to reproduce the behavior:
 
 **Additional context**
 <!--
-(Add any other context about the problem here)
+Add any other context about the problem here. E.g.: 
+1. Additional checks done with outcome, if that can help understand the issue better or pinpoint the root cause
```

Comment:
Are there alternative scenarios (different input data, different configuration/setup) in which the problem does (not) occur? If yes, please provide the steps for these scenarios.

Reply:
Done